modules/nw-anp-banp-metrics.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * list of assemblies where this module is included:
+// networking/network_security/AdminNetworkPolicy/nw-ovn-k-anp-banp-metric.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="anp-banp-metrics_{context}"]
+= Metrics
+
+[cols="1,1,1"]
+|===
+| Name | Labels |Explanation
+
+|`ovnkube_controller_admin_network_policies`
+|Not applicable
+|The total number of `AdminNetworkPolicy` in the cluster.
+
+|`ovnkube_controller_baseline_admin_network_policies`
+|Not applicable
+|The total number of `BaselineAdminNetworkPolicy` in the cluster. Values should be 0 or 1.
+
+|`ovnkube_controller_admin_network_policies_rules`
+|*`direction`: specifies either `Ingress` or `Egress`.
```

Comment:
Are these asterisks supposed to be rendering as bullets? If so, you'd need to add an "a" next to the column number above that you want to render AsciiDoc